### DISCUSSION TITLE
LAB 06.01 - Task 3

### DISCUSSION THREAD

**Discussion by ThomasPineda** [rramosp/ai4eng.v1#211] [06 - MÉTODOS NO SUPERVISADOS]
Hola, tengo un problema con la tarea 3, ya hice la función que me saca los clusters, cuantas empresas hay en cada cluster y el mean_sync_move por cluster, pero cada vez que corro la función (con la misma cantidad de clusters) me cambia los clusters y cuando hago que me la califique aparece esto 
![image](https://user-images.githubusercontent.com/86691796/213785465-c5b2a32a-b6c3-4657-a097-5cf884243e43.png)
![image](https://user-images.githubusercontent.com/86691796/213785524-111ec414-20fc-4a1a-b95c-0f2600e5f8ff.png)

Según yo este algoritmo con los mismos datos y los mismos clusters deberia siempre arrojar el mismo resultado, o me equivoco?

**Adunto mi collab para que puedan ver el código:**
https://colab.research.google.com/github/rramosp/ai4eng.v1/blob/main/content/LAB%2006.01%20-%20CLUSTERING.ipynb

**Comment by MarcosArangoTamayo** [comment]
Hola, me ocurre lo mismo. Cada clusterización es distinta. 
¿Llegaste a la solución?

**Comment by ThomasPineda** [comment]
lo resolvi cambiando mi función de signed() de esta:
![image](https://user-images.githubusercontent.com/86691796/224109591-1dbbeccb-52f2-4931-9409-80ec5e03848c.png)
 a esta: 

![image](https://user-images.githubusercontent.com/86691796/224109693-334c67cd-3299-4acc-9519-ad6d1d20a5ab.png)
